### PR TITLE
helm/kustomize: fix configmaps 

### LINKED ISSUE #103: Helm install fails due to missing config file
You get an error below with 

```bash
helm install topology-aware --namespace kube-system deployment/helm/resource-management-policies/topology-aware/
```

```console
F0814 16:13:04.334421       1 log.go:486] failed to create resource manager: resource-manager: failed to load fallback configuration /etc/nri-resource-policy/nri-resource-policy.cfg: config error: failed to apply configuration from file: config error: failed to read file "/etc/nri-resource-policy/nri-resource-policy.cfg": open /etc/nri-resource-policy/nri-resource-policy.cfg: no such file or directory
```

### PR DESCRIPTION
**Pull request by poussa**
Fixes #103

### CODE REVIEW

**Comment by fmuyassarov** [comment]
Thanks. Can you please fix the same issue for kustomize configMaps too?
For example:
- balloons: https://github.com/containers/nri-plugins/tree/main/deployment/overlays/balloons
- topology-aware: https://github.com/containers/nri-plugins/tree/main/deployment/overlays/topology-aware

File: deployment/helm/resource-management-policies/topology-aware/values.yaml [hunk 1]
```diff
@@ -9,8 +9,11 @@ image:
   pullPolicy: Always
 
 config:
-  ReservedResources:
-    cpu: 750m
+  policy:
+    ReservedResources:
+      cpu: 750m
+  logger:
+    Debug: nri-resource-policy,resource-manager,cache,policy,topology-hints
```
Comment (fmuyassarov) [inline deployment/helm/resource-management-policies/topology-aware/values.yaml:16]: What is this for ? I mean we don't have it for balloons by default, what''s the motivation to have logger enabled by default?

**Review by fmuyassarov — COMMENTED** [review]
Looks good, apart from one question.

File: deployment/helm/resource-management-policies/topology-aware/values.yaml (hunk 1 above)
Comment (poussa) [reply deployment/helm/resource-management-policies/topology-aware/values.yaml:16]: ah, that is a mistake. I'll fix.

Comment (poussa) [reply deployment/helm/resource-management-policies/topology-aware/values.yaml]: fixed

**Review by fmuyassarov — APPROVED** [review]
LGTM

**Pull request merged**